cmd/util/ledger/migrations/change_contract_code_migration.go

```diff
@@ -284,7 +284,7 @@ func NewSystemContractsMigration(
 ) *ChangeContractCodeMigration {
 	migration := NewChangeContractCodeMigration(chainID, log, rwf)
 	for _, change := range SystemContractChanges(chainID, options) {
-		migration.RegisterContractChange(change)
+		migration.registerContractChange(change)
```

Comment:
Why un-export this function?

Reply:
Wanted to enforce using `WithStagedContractUpdates` everywhere, so it encourages setting the contracts at the time of creating the migration (e.g: before calling `InitMigration`, etc.). It's not a must, but more of a best practice. Also helps with having cleaner logs.